languages/de_DE.php

```diff
@@ -1,74 +1,74 @@
 <?php
 $lang = array();
 
-$lang['user_blocked'] = "Dein Benutzer ist im System aktuell gesperrt.";
-$lang['user_verify_failed'] = "Captcha-Code ist ungültig.";
+$lang['user_blocked'] = "Ihr Benutzer ist im System gesperrt.";
+$lang['user_verify_failed'] = "Der Captcha-Code ist ungültig.";
 
-$lang['email_password_invalid'] = "E-Mail-Adresse / Passwort ungültig.";
-$lang['email_password_incorrect'] = "E-Mail-Adresse / Passwort nicht korrekt.";
+$lang['email_password_invalid'] = "E-Mail-Adresse und/oder Passwort sind ungültig.";
+$lang['email_password_incorrect'] = "E-Mail-Adresse und/oder Passwort sind falsch.";
 $lang['remember_me_invalid'] = "Das Angemeldet bleiben-Feld ist ungültig.";
 
-$lang['password_short'] = "Passwort ist zu kurz.";
-$lang['password_weak'] = "Passwort ist zu einfach.";
-$lang['password_nomatch'] = "Passwörter stimmen nicht überein";
-$lang['password_changed'] = "Passwort wurde erfolgreich geändert.";
-$lang['password_incorrect'] = "Aktuelles Passwort ist nicht korrekt.";
-$lang['password_notvalid'] = "Passwort ist ungültig.";
-
-$lang['newpassword_short'] = "Neues Passwort ist zu kurz.";
-$lang['newpassword_long'] = "Neues Passwort ist zu lang.";
-$lang['newpassword_invalid'] = "Das neue Passwort muss mindestens einen Großbuchstaben und einen Kleinbuchstaben enthalten sowie eine Ziffer.";
-$lang['newpassword_nomatch'] = "Neue Passwörter stimmen nicht überein.";
-$lang['newpassword_match'] = "Neues Passwort ist das gleiche wie das alte.";
-
-$lang['email_short'] = "E-Mail-Adresse ist zu kurz.";
-$lang['email_long'] = "E-Mail-Adresse ist zu lang.";
-$lang['email_invalid'] = "E-Mail-Adresse ist ungültig.";
-$lang['email_incorrect'] = "E-Mail-Adresse ist nicht korrekt.";
+$lang['password_short'] = "Das Passwort ist zu kurz.";
+$lang['password_weak'] = "Das Passwort ist zu einfach.";
+$lang['password_nomatch'] = "Die Passwörter stimmen nicht überein.";
+$lang['password_changed'] = "Ihr Passwort wurde erfolgreich geändert.";
```

Comment:
Hier würde ich auch "Das Passwort" schreiben. Sonst müssten die Texte darüber und darunter auch angepasst werden, zB. in "Ihre Passwörter stimmen nicht überein."